judo-runtime-core-security-keycloak/src/main/java/hu/blackbelt/judo/runtime/core/security/keycloak/KeycloakAdminClient.java

```diff
@@ -91,11 +84,7 @@ public List<Realm> getListOfRealms() {
                 .type(MediaType.APPLICATION_FORM_URLENCODED_TYPE)
                 .path("/realms")
                 .get(List.class);
-
-        return response.stream()
-                .map(r -> MapProxy.builder(Realm.class).withMap(r.entrySet().stream()
-                        .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue() instanceof List ? ECollections.asEList((List) e.getValue()) : e.getValue()))).newInstance())
-                .collect(Collectors.toList());
+        return response.stream().map(r -> KeycloakObjectMapper.objectMapper().convertValue(r, Realm.class)).collect(Collectors.toList());
```

Comment:
ObjectMapper haven't be created in every call. It can be a private field to be reusable. Except, KeycloakObjectMapper serving singleton for the call of objectMapper() method.

Reply:
singelton added in meta-keycloak